lang/cs/Org.Apache.REEF.Driver/Constants.cs

```diff
@@ -32,12 +31,6 @@ public static class Constants
         /// The class hierarchy file from .NET.
         /// </summary>
         [Obsolete("Deprecated in 0.14. Will be removed in 0.15.")]
-        public const string ClassHierarachyBin = "clrClassHierarchy.bin";
-
-        /// <summary>
-        /// The class hierarchy file from .NET.
-        /// </summary>
-        [Obsolete("Deprecated in 0.14. Will be removed in 0.15.")]
         public const string ClassHierarchyBin = "clrClassHierarchy.bin";
```

Comment:
The fields in question are unused, so it's a matter of deleting them.
I'll do this as part of the merge of this PR, as I don't believe they are worth a separate PR.